chargeio/infrastructure/CustomDateTimeConverter.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using Newtonsoft.Json.Converters;
+
+namespace chargeio
+{
+    class CustomDateTimeConverter : IsoDateTimeConverter
```

Comment:
Need to reconcile CustomDateTimeConverter vs Mapper.IsoDateConverter. Why are both present?